### ISSUE TITLE
Listing active mounts on AIX does not give valid output with NFS mount w/o options

### ISSUE THREAD

**Issue by jose1711** [saltstack/salt#46392]
### Description of Issue/Question

mounts.active run against AIX minion with NFS mounted filesystems w/o options is treated as a local filesystem and returns invalid data.

### Setup
```
# ssh aix_minion
# mount remote_host:/mnt /mnt  # nfs share mount from remote_host
# mount -p | tail -2
/dev/lv01  /testlv  jfs2  jfs2   Mar 04 17:55 rw,log=INLINE
remote_host /mnt        /mnt             nfs3   Mar 06 11:33
```

### Steps to Reproduce Issue
```
# salt-call --local mount.active
    /testlv:
        ----------
        device:
            /dev/lv01
        fstype:
            jfs2
        opts:
            - rw
            - log=INLINE
    /mnt:
        ----------
        device:
            remote_host
        fstype:
            /mnt
        opts:
            - 11:33
```

The output for /mnt is an utter garbage. Mount.py relies that ```mount -p``` always returns options in its last column - this is not always the case. I realize salt minion version we're using is not the latest one but ```active``` function inside ```mount.py``` did not change so I believe the issue is with the most recent version too.

### Versions Report
Salt Version: 2016.11.4
AIX: 7.1

**Comment by rickh563** [comment]
ZD-2311

**Comment by Ch3LL** [comment]
glancing at the code it seems this could be improved to handle a mount output show above. thanks :)

**Comment by dmurphy18** [comment]
Need to allow for remote host properly, output from mount -p, noting nfs3 mount below:

root@l490vp024_pub:/opt/freeware/lib/python2.7/site-packages/salt/modules# mount -p    
  node       mounted        mounted over    vfs       date        options      
-------- ---------------  ---------------  ------ ------------ --------------- 
         /dev/hd4         /                jfs2   May 16 16:09 rw,log=/dev/hd8 
         /dev/hd2         /usr             jfs2   May 16 16:09 rw,log=/dev/hd8 
         /dev/hd9var      /var             jfs2   May 16 16:09 rw,log=/dev/hd8 
         /dev/hd3         /tmp             jfs2   May 16 16:09 rw,log=/dev/hd8 
         /dev/hd1         /home            jfs2   May 16 16:09 rw,log=/dev/hd8 
         /dev/hd11admin   /admin           jfs2   May 16 16:09 rw,log=/dev/hd8 
         /proc            /proc            procfs May 16 16:09 rw              
         /dev/hd10opt     /opt             jfs2   May 16 16:09 rw,log=/dev/hd8 
         /dev/livedump    /var/adm/ras/livedump jfs2   May 16 16:09 rw,log=/dev/hd8 
192.168.1.11 /stage/middleware /stage/middleware nfs3   May 16 16:11 ro,bg,hard,intr,sec=sys

Need to update code to allow for empty opts

**Comment by rallytime** [comment]
@jose1711 This has been fixed with #47956. Do you want to give that a try?
